### ISSUE TITLE
Node.JS Server doesn't start

### ISSUE THREAD

**Issue by HenrikZabel** [mrousavy/react-native-mmkv#284]
When I install the package with <a href="https://github.com/mrousavy/react-native-mmkv/blob/master/INSTALL.md">this</a> tutorial I cannot start the node.js server (opens the window for a second and than closes; I get no error at all). When I try running `npx react-native run-android` it's going to give me that output:
```
C:\NUTZER\App\FoodDeliveryApp\FoodDeliveryApp>npx react-native run-android
info Running jetifier to migrate libraries to AndroidX. You can disable it using "--no-jetifier" flag.
Jetifier found 1277 file(s) to forward-jetify. Using 6 workers...
info Starting JS server...
info Launching emulator...
info Successfully launched emulator.
info Installing the app...

> Configure project :react-native-mmkv
MMKV: node_modules/ found at: C:\NUTZER\App\FoodDeliveryApp\FoodDeliveryApp\node_modules

> Task :react-native-mmkv:externalNativeBuildDebug
Build reactnativemmkv_armeabi-v7a
ninja: Entering directory `C:\NUTZER\App\FoodDeliveryApp\FoodDeliveryApp\node_modules\react-native-mmkv\android\.cxx\cmake\debug\armeabi-v7a'
Build reactnativemmkv_arm64-v8a
ninja: Entering directory `C:\NUTZER\App\FoodDeliveryApp\FoodDeliveryApp\node_modules\react-native-mmkv\android\.cxx\cmake\debug\arm64-v8a'
Build reactnativemmkv_x86
ninja: Entering directory `C:\NUTZER\App\FoodDeliveryApp\FoodDeliveryApp\node_modules\react-native-mmkv\android\.cxx\cmake\debug\x86'
Build reactnativemmkv_x86_64
ninja: Entering directory `C:\NUTZER\App\FoodDeliveryApp\FoodDeliveryApp\node_modules\react-native-mmkv\android\.cxx\cmake\debug\x86_64'

> Task :app:installDebug
Installing APK 'app-debug.apk' on 'Pixel_4_XL_API_29(AVD) - 10' for app:debug
Installed on 1 device.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.9/userguide/command_line_interface.html#sec:command_line_warnings

BUILD SUCCESSFUL in 38s
330 actionable tasks: 11 executed, 319 up-to-date
info Connecting to the development server...
8081
info Starting the app on "emulator-5554"...
Starting: Intent { cmp=com.fooddeliveryapp/.MainActivity }
```

Now I tried to reinstalled it, updated every Android Studio dependency and package, ran `cd Android` and than `gradlew clean` and also `gradlew buildDebug` but it didn't help.

This are my dependencies
```
"dependencies": {
    "@gorhom/bottom-sheet": "^4.1.5",
    "@react-native-async-storage/async-storage": "^1.15.14",
    "@react-native-clipboard/clipboard": "^1.9.0",
    "@react-native-community/blur": "^3.6.0",
    "@react-native-community/netinfo": "^7.1.7",
    "@react-navigation/bottom-tabs": "^6.0.9",
    "@react-navigation/native": "^6.0.6",
    "@react-navigation/native-stack": "^6.2.5",
    "@react-navigation/stack": "^6.0.11",
    "link-preview-js": "^2.1.8",
    "lottie-ios": "^3.2.3",
    "react": "17.0.2",
    "react-native": "0.66.4",
    "react-native-background-timer": "^2.4.1",
    "react-native-geolocation-service": "^5.3.0-beta.4",
    "react-native-gesture-handler": "^2.1.0",
    "react-native-localize": "^2.1.6",
    "react-native-maps": "^0.29.4",
    "react-native-mmkv": "^1.6.3",
    "react-native-pinchable": "^0.2.1",
    "react-native-pull-refresh": "^1.0.0",
    "react-native-reanimated": "^2.3.1",
    "react-native-safe-area-context": "^3.3.2",
    "react-native-screens": "^3.10.1",
    "react-native-startup-time": "^2.0.0",
    "react-native-svg": "^12.1.1",
    "rn-fetch-blob": "^0.12.0"
  },
```
I also made sure to use the installation guide for projects that include `reanimated`

**Comment by mrousavy** [comment]
This doesn't seem related to react-native-mmkv - does `npx react-native start --reset-cache` work for you?

**Comment by HenrikZabel** [comment]
Actually I had to update my Node version. It seems like this library does not work with an older Node.Js version. Now it should work fine

**Comment by HenrikZabel** [comment]
So I just updated my Node.JS. It seems like this library does not work fine with an older Node.js version. So after updating it to the newest version (state 15 Jan 2022).

**Comment by HenrikZabel** [comment]
So I just updated my Node.JS. It seems like this library does not work fine with an older Node.js version. So after updating it to the newest version (state 15 Jan 2022) there is no build error anymore. Now I have this error: `Error: Failed to create a new MMKV instance, the native initializer function does not exist. Is the native MMKV library correctly installed? Make sure to disable any remote debugger (e.g. Chrome) to use JSI!, js engine: hermes`

**Comment by mrousavy** [comment]
I just released v2.0.0 which automatically installs itself!
